examples/guestbook/README.md

```diff
@@ -39,19 +39,19 @@ This example shows how to build a simple, multi-tier web application using Kuber
 **Table of Contents**
```

Comment:
A diagram of the application topology showing the tiers and components would be awesome if you have time.